databricks/koalas/frame.py

```diff
@@ -8469,8 +8469,13 @@ def query(self, expr, inplace=False):
                 'For argument "inplace" expected type bool, received type {}.'
                 .format(type(inplace).__name__))
 
-        sdf = self._sdf.filter(expr)
-        internal = self._internal.copy(sdf=sdf)
+        data_columns = [label[0] for label in self._internal.column_labels]
```

Comment:
If we support multi-index column later, we need to rename to fit the pandas' requirement.